src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+function Calculator() {
+  return (
+    <div className="top-div">
+      <div className="zero-div"><div className="num-item  top-num">0</div></div>
+      <div className="numerals">
+        <div className="num-item">AC</div>
```

Comment:
- [ ] Great job deleting the project boilerplates as suggested by the previous reviewer. However, I noticed you missed an important project requirement which the previous reviewer also mentioned. 
  - Kindly ensure that the `Calculator` component has at least one child component. This [tutorial might be helpful](https://medium.com/@martin.crabtree/react-js-using-children-props-c83d5b259756). 🙏 See below for more details of the requirements:
  
<img width="847" alt="Screenshot 2023-08-15 at 21 17 50" src="https://github.com/Armynerh/math_magicians/assets/107192076/938799ba-3a49-4e54-b5ef-605cd6436b87">

  
  Happy coding!😊🚀🍀